### PR TITLE
No stream metadata when you create a stream

### PR DESCRIPTION
**Pull request by avish0694**
When you create a stream in Event Store, it seems like there's no stream metadata.

For example if you create a **StreamX** and do a curl : 
curl -i -H "Accept:application/vnd.eventstore.atom+json" "http://127.0.0.1:2113/streams/StreamX/metadata" -u "admin:changeit"
Metadata of **StreamX** appears to be empty on terminal and also in the UI.

### CODE REVIEW

File: http-api/stream-metadata.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ outputFileName: index.html
 
 # Stream Metadata
 
-Every stream in Event Store has metadata associated with it. Internally, the metadata includes information such as the ACL of the stream, the maximum count and age for the events in the stream. Client code can also add information into stream metadata for use with projections or the client API.
+By default streams you create in Event Store has no metadata associated with it. Internally, the metadata includes information such as the ACL of the stream, the maximum count and age for the events in the stream. Client code can also add information into stream metadata for use with projections or the client API.
```
Comment (ChrisChinchilla) [inline http-api/stream-metadata.md]: @avish0694 "…have no…", the changes makes it plural here, so you need to change that.

Comment (ChrisChinchilla) [reply http-api/stream-metadata.md]:
```suggestion
By default streams you create in Event Store have no metadata associated with it. Internally, the metadata includes information such as the ACL of the stream, the maximum count and age for the events in the stream. Client code can also add information into stream metadata for use with projections or the client API.
```

**Comment by ChrisChinchilla** [comment]
One tiny change @avish0694 and we're good to go.

**Comment by avish0694** [comment]
@ChrisChinchilla Done. Thanks. :)

**Pull request merged**